vm/src/frame.rs

```diff
@@ -1654,7 +1654,7 @@ impl ExecutingFrame<'_> {
             bytecode::BinaryOperator::Add => vm._add(a_ref, b_ref),
             bytecode::BinaryOperator::Multiply => vm._mul(a_ref, b_ref),
             bytecode::BinaryOperator::MatrixMultiply => vm._matmul(a_ref, b_ref),
-            bytecode::BinaryOperator::Power => vm._pow(a_ref, b_ref),
+            bytecode::BinaryOperator::Power => vm._pow(a_ref, b_ref, &vm.ctx.none()),
```

Comment:
```suggestion
            bytecode::BinaryOperator::Power => vm._pow(a_ref, b_ref, &vm.ctx.none),
```
`none()` clones value but `none` doesn't